adapter/persistence/src/main/java/org/anastaklo/portal/adapter/persistence/EmployeeJpaEntity.java

```diff
@@ -0,0 +1,120 @@
+package org.anastaklo.portal.adapter.persistence;
+
+import java.time.LocalDate;
+import java.util.HashSet;
+import java.util.Set;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.Setter;
+
+@Entity
+@Table(name = "employee")
+public class EmployeeJpaEntity extends BaseJpaEntity{
+
+    @Column
+    @Getter
+    @Setter
+    private String firstName;
+
+    @Column
+    @Getter
```

Comment:
@Getter
@Setter for class-level

Reply:
@EqualsAndHashCode for all classes?

Reply:
no, we can't use `@EqualsAndHashCode` for the vast majority of our entities because I chose automatically generated by database `id`, so this `id` differs in different entity states (it will be `null` when entity is in `transient` state and later it will be set to some number after persisting)